app/screens/home/search/results/header.tsx

```diff
@@ -78,6 +82,15 @@ const Header = ({
         onTabSelect('files');
     }, [onTabSelect]);
 
+    const snapPoints = useMemo(() => {
+        return [
+            bottomSheetSnapPoint(
+                NUMBER_FILTER_ITEMS + 1,
+                FILTER_ITEM_HEIGHT,
+                bottom) + TITLE_HEIGHT,
```

Comment:
I am not a big fan of the indentation here, maybe like this?
```suggestion
            bottomSheetSnapPoint(
                NUMBER_FILTER_ITEMS + 1,
                FILTER_ITEM_HEIGHT,
                bottom,
            ) + TITLE_HEIGHT,
```